core/operation/src/main/java/uk/gov/gchq/gaffer/named/operation/ParameterDetail.java

```diff
@@ -37,8 +38,9 @@
     private Object defaultValue;
     private Class valueClass;
     private boolean required;
+    private List<String> options;
```

Comment:
Just thinking whether it would be worth leaving this as an unparameterised list. If say a parameter was an integer but could only be 0 - 5, having the options as strings would not be compatible.

Reply:
Done, generified types input on list and added a new test in ParameterDetailTest to check it works